extensions/vscode/webviews/homeView/src/vscode.ts

```diff
@@ -0,0 +1,10 @@
+import { WebviewApi } from "vscode-webview";
+
+let apiInstance: WebviewApi<unknown> | undefined = undefined;
+
+export const vscodeAPI = (): WebviewApi<unknown> => {
```

Comment:
This was a bit weird. I tried to get `vitest` to play nicely with `acquireVsCodeApi()`, but I just couldn't get anything to work. 

So instead I created this function / singleton to call it so it was easily mockable as a module import function.